src/mongocxx/exception/server_error_code.cpp

```diff
@@ -12,26 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <mongocxx/exception/private/error_code.hpp>
-
-#include <system_error>
-
-#include <mongocxx/exception/private/error_category.hpp>
+#include <mongocxx/exception/server_error_code.hpp>
 
 #include <mongocxx/config/private/prelude.hpp>
 
 namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 
-///
-/// Translate a mongocxx::error_code into a std::error_code.
-///
-/// @param error A mongocxx::error_code
-///
-/// @return A std::error_code
-///
-std::error_code make_error_code(mongocxx::error_code error) {
-    return std::error_code(static_cast<int>(error), mongocxx_error_category());
+namespace {
+
+//
+// An error_category for codes returned by the server.
```

Comment:
Ultimately, this error space will only reflect errors returned by the server. When we are able to distinguish them, I plan to roll the mongoc local errors into the mongocx::error_code enumeration.